### DISCUSSION TITLE
[Question] Only save related ids in one collection

### DISCUSSION THREAD

**Discussion by MattJarman** [jenssegers/laravel-mongodb#2088] [General] [answered]
Hey,

I have a collection of 70,000 games, and want to avoid saving a record of all `user_ids` in every single `Game` document, but do want to keep a record of `game_ids` in each `User` document. Is there a way to achieve this?

**Comment by divine** [comment, marked as answer]
Hi,

Please see similar question here https://github.com/jenssegers/laravel-mongodb/issues/2049

Thanks!

**Comment by MattJarman** [comment]
Thank you!
